pkg/securityscan/job/job.go

```diff
@@ -21,28 +21,34 @@ import (
 const (
 	defaultTerminationGracePeriodSeconds = int64(0)
 	defaultBackoffLimit                  = int32(0)
+	defaultTTLSecondsAfterFinished       = int32(0)
 )
 
 var (
 	ConditionComplete = condition.Cond(batchv1.JobComplete)
 	ConditionFailed   = condition.Cond(batchv1.JobFailed)
 
-	BackoffLimit = func(defaultValue int32) int32 {
-		if str, ok := os.LookupEnv("SYSTEM_UPGRADE_JOB_BACKOFF_LIMIT"); ok {
-			if i, err := strconv.ParseInt(str, 10, 32); err != nil {
-				logrus.Errorf("failed to parse $%s: %v", "SYSTEM_UPGRADE_JOB_BACKOFF_LIMIT", err)
-			} else {
-				return int32(i)
-			}
-		}
-		return defaultValue
-	}(defaultBackoffLimit)
+	BackoffLimit = readFromEnv("CIS_JOB_BACKOFF_LIMIT", defaultBackoffLimit)
```

Comment:
This shouldn't be exported

Reply:
Do we not support updating this on the fly? If you grab the env on job create it could be updated without requiring a restart of the operator

Reply:
no we don't support on the fly atm, can log an enhancement ticket to change and test. Will not export it.

Reply:
https://github.com/rancher/cis-operator/issues/81

Reply:
updated to not export